vmware_aria_operations_integration_sdk/container_versions.json

```diff
@@ -2,7 +2,7 @@
     "base_image": {
         "language": "Python",
         "path": "base-python-adapter",
-        "version": "0.11.0"
+        "version": "0.12.0"
```

Comment:
Have you tested to see if the old schema will work with this server? We will definitely want to do that, as if it does not we need to make sure we're very clear that this is a breaking change.

That said, I _expect_ that the old schema will work - I just don't know with certainty.